src/utils/time.ts

```diff
@@ -169,6 +177,43 @@ export const formatHourAndMinute = (utcSeconds: number) => {
   return format(utcSeconds * ONE_SECOND_IN_MILLISECONDS, "HH:mm");
 };
 
+// TODO: do we need it?
+export const formatHour = (utcSeconds: number) => {
+  const date = new Date(utcSeconds * ONE_SECOND_IN_MILLISECONDS);
+  const hh = String(date.getHours()).padStart(2, "0");
+  return hh;
+};
+
+// TODO: do we need it?
+export const getMinutes = (utcSeconds: number) => {
+  const date = new Date(utcSeconds * ONE_SECOND_IN_MILLISECONDS);
+  const hh = date.getHours();
+  const mm = date.getMinutes();
+  const minutes = hh * 60 + mm;
+  return minutes;
+};
+
+export const getSecondsFromStartOfDay = (utcSeconds: number) => {
+  const utcMilliseconds = utcSeconds * ONE_SECOND_IN_MILLISECONDS;
+
+  return (
+    getHours(utcMilliseconds) * ONE_HOUR_IN_SECONDS +
+    getLibMinutes(utcMilliseconds) * ONE_MINUTE_IN_SECONDS +
+    getSeconds(utcMilliseconds)
+  );
+};
+
+export const isLiveEvent = (
+  startUtcSeconds: number,
+  minutesDuration: number
+) => {
+  const now = getCurrentTimeInUTCSeconds();
+  return (
+    startUtcSeconds <= now &&
+    now <= startUtcSeconds + minutesDuration * ONE_MINUTE_IN_SECONDS
+  );
+};
```

Comment:
Can we do something like this?

- https://date-fns.org/v2.20.1/docs/addMinutes
- https://date-fns.org/v2.20.1/docs/isWithinInterval

```suggestion
export const isLiveEvent = (
  startUtcSeconds: number,
  minutesDuration: number
) => {
  const start = startUtcSeconds * ONE_SECOND_IN_MILLISECONDS;
  const end = addMinutes(start, minutesDuration);
  
  return isWithinInterval(Date.now(), { start, end });
};
```

Reply:
yeah, nice! 
fixed: https://github.com/sparkletown/sparkle/pull/1302/commits/4620ee606b4d082901e989fc6eac27f0e5b9a265